### ISSUE TITLE
Include IAM policy in readme

### ISSUE THREAD

**Issue by shsu** [silinternational/ecs-deploy#26]
It would be helpful for other users if we include a sample IAM policy that grants the necessary permissions for this tool. This is what I have currently:

```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "Stmt1457037856137",
            "Action": [
                "ecs:DeregisterTaskDefinition",
                "ecs:DescribeServices",
                "ecs:DescribeTaskDefinition",
                "ecs:DescribeTasks",
                "ecs:ListTasks",
                "ecs:RegisterTaskDefinition",
                "ecs:StartTask",
                "ecs:StopTask",
                "ecs:UpdateService"
            ],
            "Effect": "Allow",
            "Resource": "*"
        }
    ]
}
```

**Comment by diddeb** [comment]
[Here's](https://github.com/silinternational/ecs-deploy/pull/28) the PR.

**Comment by shsu** [comment]
:+1:

**Comment by fillup** [comment]
I merged in #28
